platform-includes/getting-started-primer/javascript.nuxt.mdx

```diff
@@ -2,3 +2,36 @@
   This SDK is currently in **beta**. Beta features are still in progress and may have bugs. Please reach out on
   [GitHub](https://github.com/getsentry/sentry-javascript/issues/new/choose) if you have any feedback or concerns.
 </Alert>
+
+
+Sentry's Nuxt SDK enables automatic reporting of errors and performance data.
+
+## Compatibility
+
+The Sentry Nuxt SDK supports Nuxt version `3.7.0` and above. For best results, we recommend
+using Nuxt `3.14.0` or later, which includes updated dependencies critical to the SDK's functionality.
+
+In case you are using Nuxt before version `3.14.0`, add the following overrides:
+
+```json {tabTitle:npm} {filename:package.json}
+"overrides": {
+  "ofetch": "^1.4.0"
+  "@vercel/nft": "^0.27.4"
+}
+```
+
+```json {tabTitle:yarn} {filename:package.json}
+"resolutions": {
+  "ofetch": "^1.4.0"
+  "@vercel/nft": "^0.27.4"
+}
+```
+
+```json {tabTitle:pnpm} {filename:package.json}
+"pnpm": {
+  "overrides": {
+    "ofetch": "^1.4.0"
+    "@vercel/nft": "^0.27.4"
+  }
+}
+```
```

Comment:
I'd remove all of this from the getting started page as the wizard takes care of it and I think it's just confusing for users. It's better kept in imanual installation methods.

Reply:
I removed everything after "In case you are using Nuxt before version `3.14.0`, add the following overrides:" 👍 

I left the general version compatibility notice.